docs/build/arm64ec-windows-abi-conventions.md

```diff
@@ -50,7 +50,7 @@ Special helper routines like `__chkstk_arm64ec` use custom calling conventions a
 | `x15` | `mm7` (low 64 bits of x87 `R7` register) | volatile | volatile | volatile |
 | `x16` | High 16 bits of each of the x87 `R0`-`R3` registers | volatile(`xip0`) | volatile(`xip0`) | volatile |
 | `x17` | High 16 bits of each of the x87 `R4`-`R7` registers | volatile(`xip1`) | volatile(`xip1`) | volatile |
-| `x18` | N/A | fixed(TEB) | fixed(TEB) | volatile |
+| `x18` | GS.base | fixed(TEB) | fixed(TEB) | N/A |
```

Comment:
Should be

| `x18` | GS.base | fixed(TEB) | fixed(TEB) | fixed(TEB) |

Reply:
fixed - thank you!